src/components/ModelTableList/ModelSummary/ModelSummary.tsx

```diff
@@ -28,7 +28,7 @@ const ModelSummary = ({ modelData, ownerTag }: Props) => {
       <Tooltip
         message="See applications"
         position="top-center"
-        className="u-flex--block has-icon"
+        className="u-flex--block u-has-icon"
```

Comment:
I think there might be some other places where the same change should be performed e.g. `SearchHistoryMenu`, `SecretsTable`, `SecrtetsPicker`, `ShareModel`.

Reply:
In those other places the `has-icon` is applied to a button, where using `has-icon` is still the right class name (https://vanillaframework.io/docs/patterns/buttons#icon) whereas `u-has-icon` is for non-button elements (https://github.com/canonical/vanilla-framework/issues/3102).